web/satellite/src/components/common/ListComponent.vue

```diff
@@ -0,0 +1,35 @@
+// Copyright (C) 2019 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+<template>
+    <div>
```

Comment:
remove

Reply:
Repeatable div must be wrapped by div. 
Leaving repeatable div will cause error during render

Reply:
Nonsense, you just must give v-for div key attribute

Reply:
No it is not. You must have one root element.
Vue interprets the root element having a v-for loop on it as producing multiple root level elements.